### DISCUSSION TITLE
Unable to Login

### DISCUSSION THREAD

**Discussion by ShudarsanRegmi** [Harshdev098/Research-Nexas#291] [General]
I set up the environment and the website is running successfully. But I'm not being able to register a  new user unfortunately. 
I'm getting validation error on the UI. When I inspected, I found that  I'm getting 400 response on /create_user
![{8679E5B6-520D-46C4-A0CE-E5E9FC114A28}](https://github.com/user-attachments/assets/7a03282b-50d0-4bf9-ba28-7548990ece91)

How can I fix this error?

**Comment by Harshdev098** [comment]
Can you share the error in the terminal if it isn't fixed @ShudarsanRegmi
